client/components/Message/Actions/Actions.tsx

```diff
@@ -19,14 +19,20 @@ type ActionOptions = {
 const Actions: FC<{ actions: Array<ActionOptions>; runAction: RunAction; mid: string }> = ({
 	actions,
 	runAction,
-}) => (
-	<Content width='full' justifyContent='center'>
-		<ButtonGroup align='center'>
-			{actions.map((action) => (
-				<Action runAction={runAction} key={action.id} {...action} />
-			))}
-		</ButtonGroup>
-	</Content>
-);
+}) => {
+	const t = useTranslation();
+
+	const alignment = actions[0]?.label === t('Join_call') ? 'flex-start' : 'center';
```

Comment:
Same comment goes for here
https://github.com/RocketChat/Rocket.Chat/pull/22689/files#r672064888

Reply:
@djcruz93 You're right earlier, we definitely need to add some logic here to control the alignment but we need to make sure that its applied in a general way, so here's what I'm proposing here.

1. Introduce a new property to the actionLinks object, lets call it `actionLinksAlignment` for now. This property only needs to be set on the first action in the `actions` array and there we can specify the alignment value.
So [this](https://github.com/RocketChat/Rocket.Chat/pull/22689/files#diff-33cc1585dcb8424c6800cd440c0c3c8249a58b8ecafd74e38b39a408ac57f2c6R120-R129) would become
```
			actionLinks: {
				webrtc: [
					{ actionLinksAlignment: 'flex-start',  i18nLabel: 'Join_call', label: TAPi18n.__('Join_call'), method_id: 'joinLivechatWebRTCCall' },
					{ i18nLabel: 'End_call', label: TAPi18n.__('End_call'), method_id: 'endLivechatWebRTCCall' },
				],
				jitsi: [
					{ icon: 'icon-videocam', i18nLabel: 'Accept', method_id: 'createLivechatCall' },
					{ icon: 'icon-cancel', i18nLabel: 'Decline', method_id: 'denyLivechatCall' },
				],
			},
```
2. Then in this file, parse the value of this property and use it for alignment. Also we need to make sure that the default value is still `center`

Reply:
@murtaza98 Do you think storing frontend attributes in the database is a good idea? Also, we need to anyway resolve the actionLinksAlignment in the general component.
```javascript
const Actions: FC<{ actions: Array<ActionOptions>; runAction: RunAction; mid: string }> = ({
	actions,
	runAction,
}) => {
	const t = useTranslation();

	const alignment = actions[0]?.actionLinksAlignment || 'center';

	return (
		<Content width='full' justifyContent={alignment}>
```
Please do confirm once.

Reply:
Yups, that shouldn't be an issue.